python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/management_api_models.schema.yml

```
@@ -27,3 +31,41 @@ keys:
                   disabled:
                     type: bool
                     default: false
+      provider:
+        type: dict
+        keys:
+          sysdb:
+            type: dict
+            description: Sysdb provider configuration.
+            keys:
+              disabled_paths:
+                type: list
+                description: List of unique disabled Sysdb path for Octa.
+                items:
+                  type: str
+          smash:
+            type: dict
+            description: Smash provider configuration.
+            keys:
+              paths:
```

We can reduce one level
```suggestion
          smash_paths:
```

---

I understand your point that we currently have only one key under smash, so it makes sense to use it with smash directly. However, my point is that smash_paths is not really a provider. To align with the EOS CLI, I kept paths under smash. Let me know your thoughts. With your suggestion then we have to change sysdb as sysdb_disabled_paths.

---

I prefer the nesting, since the provider is smash. And there might come other options later.